Makefile

```
@@ -20,7 +20,7 @@ $(GOBIN): ## create bin/ in the current directory
 
 
 .PHONY: build
-build: lint ## build the terraform provider
+build: ## build the terraform provider
```

```suggestion
build: lint build-ci ## build the terraform provider

.PHONY: build-ci
build-ci:
```

Personally, I think I'd prefer `make build` to run the linter locally. WDYT?

---

Sounds good, fixed so!